Fig4 rewrite1 (#176)
Klepac-Ceraj-Lab/Resonance: manuscript/main.tex

were also oportunistic pathogens, or previously reported
to be linked with disease (eg. \textit{R. gnavus}, \textit{E. coli}).

> \subsubsection*{Gut microbial taxa relate diffrentially to distinct cognitive subscales} %TODO: add numbers!

Reviewer: ```suggestion
\subsubsection*{Gut microbial taxa are associated with specific cognitive functions} %TODO: add numbers!
```
